packages/block-editor/src/store/reducer.js

```diff
@@ -2087,6 +2087,25 @@ export function hoveredBlockClientId( state = false, action ) {
 	return state;
 }
 
+/**
+ * Reducer setting the insertion point
+ *
+ * @param {boolean} state  Current state.
+ * @param {Object}  action Dispatched action.
+ *
+ * @return {boolean} Updated state.
+ */
+export function inserterInsertionPoint( state = false, action ) {
+	switch ( action.type ) {
+		case 'SET_INSERTER_INSERTION_POINT':
+			return action.value;
+		case 'SELECT_BLOCK':
+			return null;
```

Comment:
Clear the insertion point after a new block selection.

Reply:
This doesn't seem to be working in my testing...